test/blackbox-tests/test-cases/corrections.t/run.t

```diff
@@ -93,6 +93,10 @@ One fix is to not look at it, and flag its existence as an error.
 
   $ dune build text-file-corrected
   $ dune build @correction1
+  File "text-file", line 1, characters 0-0:
+  Error: Files _build/default/text-file and _build/default/text-file-corrected
+  differ.
+  [1]
```

Comment:
It took me a while to understand why this error appeared, but I think I understand now: before this PR, the stamp for `@correction1` wasn't taken into account whether `text-file-corrected` existed or not. While now, we compute the stamp much later and in particular after we have  observed whether `text-file-corrected` exist or not. And this info ends up in the stamp. That's why the action is being re-executed.

Reply:
Scratch that. It was just a bug.